toolkit/tools/imagecustomizerapi/README.md

```diff
@@ -0,0 +1,16 @@
+# Mariner Image Customizer API
+
+The Mariner image customizer (imgcustomizer) will be released as a standalone tool and
+will provide strong backwards compatibility guarantees (after the first official
+release).
+This is contrast to the Mariner toolkit's new image config, which isn't officially
+released and doesn't provide any backwards compatibility guarantees.
+
+While currently the new image config and imgcustomizer config are very similar, in the
+future there is the possibility they will diverge.
+
+## Known differences
+
+- For the new image config, `AdditionalFiles`' source files are relative to the working
```

Comment:
Are both of these saying the same thing? Do you mean "Wheras, for the old imager tool the source files are realtive to the config file."?

Reply:
My understanding is that for the image creation tool, the paths within the config file are relative to the working directory. (Though this isn't explicitly in the code. So, I could be wrong about this.) That being said, most of the time, when the tool runs, the working directory is set to the config file parent directory.

Whereas, for imgcustomizer, I want the paths to always be relative to the config file's parent directory.